src/main/java/net/conjur/util/Args.java

```diff
@@ -88,6 +87,20 @@ public static long notNegative(final long n, final String name) {
         return n;
     }
 
+    public static String getMandatoryProperty(String name) {
```

Comment:
Are this also copied from the Apache library, or did you author these yourself?

Reply:
I did this myself, should this be moved to a separate class in utils?

Reply:
My lean is yes... just so they aren't confused with methods copied over. I would at least add a comment indicating this.